### PR TITLE
dynamically check if the missing_* columns exist

### PR DESCRIPTION
**Pull request by sgratzl**
use SQLAlchemy to check whether the missing columns (`missing_value`, ...) exists in the covidcast table. If they to, query them, if they don't provide dummy values instead. 

This could be a workaround till we have the database updated

### CODE REVIEW

File: src/server/endpoints/covidcast.py
Comment (chinandrew) [inline src/server/endpoints/covidcast.py:155]: Are we ok with setting 0 as the default value (as opposed to "not implemented" or something), since it means entry not missing, which isn't necessarily true for existing data.

Comment (sgratzl) [reply src/server/endpoints/covidcast.py:155]: what would be an alternative value?

Comment (chinandrew) [reply src/server/endpoints/covidcast.py:155]: That's....a good question. Does it make sense to not return those fields at all?

Comment (krivard) [reply src/server/endpoints/covidcast.py:155]: Prefer not returning those fields at all, unless that will continue to block a project we were hoping to unblock with this PR. How many of the things that are currently blocked on nans, actually need the missingness columns and not just a compatible database+server?

- [ ] dashboard builder
- [ ] speed improvements for frontend queries (there are piles of these; are any of them trying to explicitly exclude the missingness columns? & liable to get confused when there are no missingness columns to exclude?)
- [ ] advanced metadata
- [x] quidel revisions (need missingness annotations to handle deletions)

Comment (sgratzl) [reply src/server/endpoints/covidcast.py]: for now I changed it that the columns are not returned. I guess for the quidel revisions returning dummy values won't help this sub project either

**Review by krivard — APPROVED** [review]
👍  merge at will

**Pull request merged**
